setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+setup(
+    name='scrapy-crawlera',
+    version='1.0.1',
+    license='BSD',
+    description='Crawlera middleware for Scrapy',
+    author='Scrapinghub',
+    author_email='info@scrapinghub.com',
+    url='https://github.com/scrapinghub/scrapy-crawlera',
+    py_modules=['crawlera', 'hubproxy'],
```

Comment:
In the example `scrapinghub.crawlera` module is used, but here `crawlera` module is installed, without 'scrapinghub' package.